doc/interfaces/BUAN_flow.rst

```diff
@@ -0,0 +1,85 @@
+.. _BUAN_flow:
+
+=====================================================================================
+   Tutorial for BUndle ANalytics (BUAN) framework using DIPY Workflow command lines
+=====================================================================================
+
+This tutorial walks through the steps for reproducing Bundle Analytics results
+on Parkinson's Progression Markers Initiative (PPMI) data derivatives.
+Bundle Analytics is a framework for comparing bundle profiles and shapes of
+different groups. In this example, we will be comparing healthy controls and
+patients with parkinson's disease. We will be using PPMI data derivatives generated
+using DIPY.
+
+
+First we need to download streamline atlas of 30 bundles in MNI space from::
+
+    https://figshare.com/articles/Atlas_of_30_Human_Brain_Bundles_in_MNI_space/12089652
+
+For this tutorial we will be using a test sample of DIPY Processed Parkinson's
+Progression Markers Initiative (PPMI) Data Derivatives. It can be downloaded
+from the link below::
+
+     https://doi.org/10.35092/yhjc.12098397
+
+There are two parts of Bundle Analytics group comparison framework,
+bundle profile analysis and bundle shape similarity.
+
+-------------------------------------------------------------------
+      We first start with group comparison of bundle profiles
```

Comment:
I need to check if these spaces do not generate an error. But you can do it too! 
```
$ cd doc
$ pip install -U -r doc-requirements.txt
$ make -C . clean && make -C . html-after-examples
$ open build/html/index.html
```